ios/RCCNavigationController.m

```diff
@@ -326,8 +327,20 @@ -(void)setButtons:(NSArray*)buttons viewController:(UIViewController*)viewContro
     id icon = button[@"icon"];
     if (icon) iconImage = [RCTConvert UIImage:icon];
     
+    id systemIcon =button[@"buttonSystemItem"];
+    UIBarButtonSystemItem systemItemIcon=NSNotFound;
+    if (systemIcon) systemItemIcon =[RCTConvert UIBarButtonSystemItem:button[@"buttonSystemItem"]];
+
+    id buttonColorTag = button[@"buttonColor"];
+    UIColor *buttonColor = nil;
+    if (buttonColorTag) buttonColor= [RCTConvert UIColor:buttonColorTag];
+
+
     UIBarButtonItem *barButtonItem;
-    if (iconImage)
+    if (systemItemIcon!=NSNotFound) {
+        barButtonItem = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:systemItemIcon  target:self action:@selector(onButtonPress:)];  
```

Comment:
It looks like there are two spaces between `systemItemIcon` and `target`. Do you mind making this just one space?